packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -119,43 +116,36 @@ contract L2OutputOracle is Initializable, Semver {
 
         startingTimestamp = _startingTimestamp;
```

Comment:
Just realizing that we could pack `startinTimestamp` and `startingBlocknumber` into the same storage slot by making them uint128s, solidity will handle that automatically. I don't think it adds a ton of value really to be honest, just saves a storage slot

Reply:
Eh I don't think it really makes a difference